### DISCUSSION TITLE
Problem updating from elementary 7.0 to 71

### DISCUSSION THREAD

**Discussion by ezymel** [elementary/browser#143] [Q&A]
I am trying to figure a way to ask this question.  When I downloaded elementary 7.0, I paid for it.  Now, I want to update to 7.1.  It doesn't let me do that.  Every time I try to update, it brings me back to the pay section and that, I do not understand.  So, how can I update?  Please help and thanks.

**Comment by danirabbit** [comment]
You don't need to pay anything or download anything from the web to update between minor versions of elementary OS. You can update from OS 7.0 to OS 7.1 by opening AppCenter and installing all of the available updates there
